kimchi/src/circuits/polynomials/foreign_field_add/circuitgates.rs

```diff
@@ -189,7 +192,12 @@ where
 }
 
 // Auxiliary function to obtain the constraints to check a carry flag
-fn carry<F: PrimeField, T: ExprOps<F>>(flag: &T) -> T {
+fn is_carry<F: PrimeField, T: ExprOps<F>>(flag: &T) -> T {
     // Carry bits are -1, 0, or 1.
     flag.clone() * (flag.clone() - T::one()) * (flag.clone() + T::one())
 }
+
+// Auxiliary function to obtain the composed value of the two lowest limbs of an element
+fn two_limb<F: PrimeField, T: ExprOps<F>>(lo_limb: &T, mi_limb: &T) -> T {
```

Comment:
This might be better to call `compact_limb` it can also be reused in your witness code.